plugin.ini

```diff
@@ -7,5 +7,5 @@ link="http://omeka.org/codex/Plugins/ExhibitBuilder_3.0"
 support_link="http://omeka.org/forums/forum/plugins"
 omeka_minimum_version="2.3"
 omeka_target_version="2.3"
-version="3.2"
+version="3.2.1"
```

Comment:
This (and the upgrade version check) should probably be 3.3. This is more than a "patch" level update.